src/test/munit/publishConsume/correlation-id-pattern.xml

```diff
@@ -34,6 +34,8 @@
         <flow-ref name="purgeQueue"/>
```

Comment:
I can not leave a comment on line number 7 because this is not relative to your changes, but please could you remove the mtf namespace? Because it is never used. (lines 7 and 12)